apps/molgenis-viz/lib/main.js

```diff
@@ -0,0 +1,66 @@
+
+// display components
+import Accordion from "../src/components/display/Accordion.vue"

```

Comment:
Scares me a bit how much this overlaps with molgenis-components. So I suppose something to talk about soon hereafter.